chsdi/models/vector/vbs.py

```diff
@@ -160,3 +160,203 @@ class Waldschaden(Base, Vector):
     y_koord = Column('y_koord', Integer)
 
 register('ch.vbs.waldschadenkarte', Waldschaden)
+
+
+class sim_facilities_a(Base, Vector):
+    __tablename__ = 'sim_fac_anhoerung'
+    __table_args__ = ({'schema': 'militaer', 'autoload': False})
+    __template__ = 'templates/htmlpopup/sim_facilities.mako'
+    __bodId__ = 'ch.vbs.sachplan-infrastruktur-militaer_anhorung'
+    id = Column('stabil_id', Text, primary_key=True)
+    facility = Column('facility', Text)
+    facname_de = Column('facname_de', Text)
+    facname_fr = Column('facname_fr', Text)
+    facname_it = Column('facname_it', Text)
+    fackind_text_de = Column('fackind_text_de', Text)
+    fackind_text_fr = Column('fackind_text_fr', Text)
+    fackind_text_it = Column('fackind_text_it', Text)
+    facstatus_text_de = Column('facstatus_text_de', Text)
+    facstatus_text_fr = Column('facstatus_text_fr', Text)
+    facstatus_text_it = Column('facstatus_text_it', Text)
+    validfrom = Column('validfrom', Text)
+    description_de = Column('description_de', Text)
+    description_fr = Column('description_fr', Text)
+    description_it = Column('description_it', Text)
+    doc_web = Column('doc_web', Text)
+    objname_de = Column('objname_de', Text)
+    objname_fr = Column('objname_fr', Text)
+    objname_it = Column('objname_it', Text)
+    bgdi_created = Column('bgdi_created', Text)
```

Comment:
We have a lot of duplicated attributes. Please create a base class and extend if needed.
https://github.com/geoadmin/mf-chsdi3/blob/master/chsdi/models/vector/uvek.py#L305
for inspiration.